README.md

```diff
@@ -39,78 +39,95 @@
-$ python3 -m pip install semgrep
 
-# To try Semgrep without installation run via Docker
-$ docker run --rm -v "${PWD}:/src" returntocorp/semgrep semgrep
-```
-
-2.  Go to your app's root directory and run `semgrep scan --config auto`. This will scan your project with the default settings.
+For beginners, we recommend starting with the [Semgrep Cloud Platform](#option-2-getting-started-from-the-semgrep-cloud-platform-recommended) because it provides a visual interface, a demo project, result triaging and exploration workflows, and makes setup in CI/CD fast. Scans are still local and code isn't uploaded. Alternatively, you can also start with the CLI and navigate the terminal output to run one-off searches.
 
-3.  [Optional, but recommended] Run `semgrep login` to get the login URL for the Semgrep Cloud Platform. Open the login URL in the browser and login.
 
-### Option 2: Getting started from the Semgrep Cloud Platform (Recommended)
+### Option 1: Getting started from the Semgrep Cloud Platform (Recommended)
 
 <a href="https://go.semgrep.dev/login-ghrmgo"  target="_blank"><img src="https://raw.githubusercontent.com/returntocorp/semgrep/develop/images/semgrep-main-image.jpg" alt="Semgrep platform image"/> </a>
 
-1.  Register to <a href="https://go.semgrep.dev/login-ghrmgo" target="_blank">semgrep.dev</a>
+1.  Register on <a href="https://go.semgrep.dev/login-ghrmgo" target="_blank">semgrep.dev</a>
 
-2.  Explore the demo app
+2.  Explore the demo project
 
 3.  Scan your project by navigating to `Projects > Scan New Project > Run scan in CI`
 
-4.  Select your version control system and follow the wizard to add your project. After this setup, Semgrep will scan your project after every pull request.
+4.  Select your version control system and follow the onboarding steps to add your project. After this setup, Semgrep will scan your project after every pull request.
 
-5.  [Optional but recommended] If you want to run Semgrep locally, follow the steps in the CLI section.
+5.  [Optional] If you want to run Semgrep locally, follow the steps in the CLI section.
 
 ### Notes:
 
-1.  Visit [Docs > Running rules](https://semgrep.dev/docs/running-rules/) to learn more about `auto` config and other rules.
+1. If there are any issues, <a href="https://go.semgrep.dev/slack" target="_blank">please ask for help in the Semgrep Slack</a>
 
-2.  If there are any issues, <a href="https://go.semgrep.dev/slack" target="_blank">please ask for help in the Semgrep Slack</a>
 
-3.  To run Semgrep Supply Chain, [contact the Semgrep team](https://semgrep.dev/contact-us).
-    Visit the [full documentation](https://semgrep.dev/docs/getting-started/) to learn more.
+### Option 2: Getting started from the CLI
+
+1.  Install Semgrep CLI
+
+```
+# For macOS
+$ brew install semgrep
+
+# For Ubuntu/WSL/Linux/macOS
+$ python3 -m pip install semgrep
+
+# To try Semgrep without installation run via Docker
+$ docker run --rm -v "${PWD}:/src" returntocorp/semgrep semgrep
```

Comment:
How do you use Semgrep login in conjunction with the docker method?

Reply:
Via passing the interactive terminal flag and then by passing in the generated app token.

```
$ docker run -it -v "${PWD}:/src" returntocorp/semgrep semgrep login
$ docker run -e SEMGREP_APP_TOKEN=<TOKEN> --rm -v "${PWD}:/src" returntocorp/semgrep semgrep ci
```